src/configuration/app/variables.md

```diff
@@ -0,0 +1,115 @@
+# Variables
+
+Platform.sh provides a number of ways to set [environment variables](/development/variables.md), either globally or specific to a single environment.  For values that should be consistent between different environments (because they're configuring the application or runtime itself, generally) the easiest way to control them is to set them in the `.platform.app.yaml` file.
```

Comment:
Yes, I was referring to the first line. It should read "Platform.sh provides a number of ways to set variables".